dash_bio/NeedlePlot.py

```diff
@@ -5,30 +5,32 @@
 
 class NeedlePlot(Component):
     """A NeedlePlot component.
-
+The Needle Plot component is used to display large datasets
```

Comment:
@shammamah I wonder if it would make more sense to write "used to **visualize** large datasets" rather than "_display_ large datasets" here?